usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -186,24 +198,33 @@ def verify_delta_table_loaded_to_delta(
     else:
         expected_table_name = delta_table_name.split(".")[-1]
 
-    partition_col = TABLE_SPEC[delta_table_name].get("partition_column")
+    # Resolve Column(s) to sort on in order to compare results
+    partition_column = TABLE_SPEC[delta_table_name].get("partition_column")
+    if partition_column:
```

Comment:
Is `partition_column` a Boolean? If not I think this if statement could be made more clear.

Reply:
Agreed. Updated to include `is not None`